server/src/main/java/org/opensearch/index/compositeindex/Metric.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.index.compositeindex;
```

Comment:
Aren't these data-cube specific within composite index? Should anything apart from a field list be a part of sub-package `org.opensearch.index.compositeindex.datacube`?

Reply:
you're right , but I was conflicted whether star tree then should be under `compositeindex.startree` package or `compositeindex.datacube.startree`

I think later to be consistent ?